docs/topics/downloader-middleware.rst

```diff
@@ -1074,6 +1074,21 @@ implementing the methods described below.
 .. autoclass:: RobotParser
    :members:
 
+RobotsTxtMiddleware Settings
```

Comment:
The setting should only be described in one page, probably the settings page. In this page you can instead mention that the user agent that the middleware uses may be overridden with this setting, providing a link to the settings page entry. See how `ROBOTSTXT_OBEY` is referenced in this page but its documentation is in the settings page only.